subsys/bootloader/bl_storage/Kconfig

```diff
@@ -6,4 +6,4 @@
 
 config SECURE_BOOT_STORAGE
 	bool "Library for accessing the bootloader storage"
-	select NRFX_RRAMC if SOC_SERIES_NRF54LX
+	select NRFX_RRAMC if SOC_SERIES_NRF54LX && !TRUSTED_EXECUTION_NONSECURE
```

Comment:
> Is this still needed as you guarded select SECURE_BOOT_STORAGE with if !TRUSTED_EXECUTION_NONSECURE?

Since Nonsecure does not support NRFX_RRAMC at all, I think that it is a good idea to explicitly make sure that it is never selected for the non-secure case.

> Maybe even the whole SECURE_BOOT_STORAGE Kconfig option should depend on !TRUSTED_EXECUTION_NONSECURE?

bl_storage is called from tf-m functions such as in attest_hal.c, so it must be available.